homeassistant/components/aqara/__init__.py

```diff
@@ -0,0 +1,205 @@
+"""Support for Aqara Smart devices."""
+from __future__ import annotations
+
+import logging
+from typing import NamedTuple
+
+from aqara_iot import (  # AqaraDevice,
```

Comment:
Stale/commented code?

```suggestion
from aqara_iot import (
```